person.rb

```diff
@@ -7,9 +7,10 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [  OPTIONAL ] Hi there! 🌟 It's widely considered a best practice to place Setter and Getter methods at the top of the class structure. 😊 Currently, they're located within the body of the class, which might make the code more organized if we move them up to the top. 🚀 This approach enhances code readability and makes it easier for fellow developers to understand and work with the class. 📜🤓